lib/Cake/Test/Case/View/Helper/HtmlHelperTest.php

```diff
@@ -553,6 +553,31 @@ public function testImageTagWithTheme() {
 		}
 	}
 
+	/**
+	 * testBase64ImageTag method
+	 *
+	 * @return void
+	 */
+	public function testBase64ImageTag()
+	{
+		$this->Html->request->webroot = '';
+		
+		$result = $this->Html->image('cake.icon.png', ['base64' => true]);
```

Comment:
tbh I like it to be one method. One can simply put a few switches and have inline or referenced (traditional src) images <3.